charts/k8ssandra/templates/stargate/stargate.yaml

```diff
@@ -111,11 +111,13 @@ spec:
               path: /checker/liveness
               port: health
             timeoutSeconds: 10
-            initialDelaySeconds: 240
+            initialDelaySeconds: {{ .Values.stargate.livenessInitialDelaySeconds | default 30 }}
```

Comment:
Since you also define the default in `values.yaml` the `default` function is unnecessary here. `values.yaml` is part of the chart just like template files. If someone or another charts uses the k8ssandra charts and supplies their own values either by way of command line args with `--set` and/or with a chart properties file, they will be merged with `k8ssandra/values.yaml`. This means that `livenessInitialDelaySeconds` and `readinessInitialDelaySeconds` will not be unset for example by using a new chart properties file.

Does it make sense to make the other probe properties configurable as well? I think it is a question worth asking, but I haven't done enough research to see what is done elsewhere to formulate an opinion. 

I will say this though. It would be nice if we could expose probes in a consistent way across components, although that could be difficult due to differences within components.

Reply:
> Since you also define the default in values.yaml the default function is unnecessary here. values.yaml is part of the chart just like template files.

Gotcha, will change that, thanks!

> Does it make sense to make the other probe properties configurable as well?

We'd need to weigh the pros (increased configurability) and cons (increased configuration complexity). My personal position would be to wait until more configurability is actually requested by users.

Reply:
>We'd need to weigh the pros (increased configurability) and cons (increased configuration complexity). My personal position would be to wait until more configurability is actually requested by users.

👍